webclient/src/modules/interactive-map.js

```diff
@@ -211,11 +211,39 @@ navigatum.registerModule("interactive-map", (function() {
             const nav = new mapboxgl.NavigationControl();
             map.addControl(nav, 'top-left');
             
-            // Fullscreen currently only on mobile
-            if (window.matchMedia &&
-                window.matchMedia("only screen and (max-width: 480px)").matches) {
-                map.addControl(new mapboxgl.FullscreenControl());
+            // (Browser) Fullscreen is enabled only on mobile, on desktop the map
+            // is maximized instead. This is determined once to select the correct
+            // container to maximize, and then remains unchanged even if the browser
+            // is resized (not relevant for users but for developers).
+            const is_mobile = window.matchMedia &&
+                              window.matchMedia("only screen and (max-width: 480px)").matches;
+
+            fs_control = new mapboxgl.FullscreenControl({
+                container: is_mobile ? document.getElementById("interactive-map")
+                                     : document.getElementById("interactive-map-container")
+            });
+            fs_control.__onClickFullscreen = fs_control._onClickFullscreen;
+            fs_control._onClickFullscreen = function() {
```

Comment:
What is de distinction between `__onClickFullscreen` and `_onClickFullscreen`?
Is there a better way to name this? Seems an easy source for future errors..

Reply:
This is just the "backup" of the mapboxgl default fullscreen handler, that I extended by overwriting this function. I've renamed it to make it clearer.